### PR TITLE
Foosball website

### PR DESCRIPTION
**Pull request by MarijusSmig**

### CODE REVIEW

File: Foosball/FoosballWebsite/Models/Match.cs [hunk 1]
```diff
@@ -5,15 +5,46 @@ namespace FoosballWebsite.Models
...
+            get{ return _GuestScore; }
+            set
+            {
+                if (_GuestScore != value)
+                    _IsChangedGuest = true;
+
+                _GuestScore = value;
+            }
+        }
+        public bool IsChangedHost {
```
Comment (narusevic) [inline Foosball/FoosballWebsite/Models/Match.cs]: public bool IsChangedHost 
{

File: Foosball/FoosballWebsite/Models/Match.cs [hunk 2]
```diff
@@ -5,15 +5,46 @@ namespace FoosballWebsite.Models
 {
     public class Match : IEntityBase
     {
+        private bool _IsChangedHost = false;
```
Comment (narusevic) [inline Foosball/FoosballWebsite/Models/Match.cs]: _isChangedHost

File: Foosball/FoosballWebsite/app/shared/components/chat.component.ts [hunk 3]
```diff
@@ -10,7 +10,6 @@ import { FeedService } from '../services/feed.service';
 export class ChatComponent implements OnInit {
 
     @Input() matches: Match[];
-    @Input() connection: string;
```
Comment (narusevic) [inline Foosball/FoosballWebsite/app/shared/components/chat.component.ts:13]: why is this deleted?

**Review by narusevic — APPROVED** [review]

File: Foosball/FoosballWebsite/app/shared/components/chat.component.ts (hunk 3 above)
Comment (MarijusSmig) [reply Foosball/FoosballWebsite/app/shared/components/chat.component.ts:13]: it only displays user connection string which is unnecesary
